### PR TITLE
update Key caching examples

### LINKED ISSUE #58: Key header modifier r
http://igrigorik.github.io/http-client-hints/
2.2.2   Interaction with Caches
uses the `Key` header modifier `r`. However http://tools.ietf.org/html/draft-fielding-http-key-02 does not define this modifier. Is there a more recent version of the draft to be listed in '11. Normative References'?

### PR DESCRIPTION
**Pull request by igrigorik**
New spec: https://tools.ietf.org/html/draft-ietf-httpbis-key-00

Closes #58.

/cc @mnot @yoavweiss.. please sanity check :)

### CODE REVIEW

**Comment by yoavweiss** [comment]
LGTM

File: draft.md [hunk 1]
```diff
@@ -116,25 +116,25 @@ Above example indicates that the cache key should be based on the DPR header.
 
 Above example indicates that the cache key should be based on the DPR, Width, and Downlink headers.
 
-Client Hints MAY be combined with Key ({{I-D.fielding-http-key}}) to enable fine-grained control of the cache key for improved cache efficiency. For example, the server MAY return the following set of instructions:
+Client Hints MAY be combined with Key ({{I-D.httpbis-http-key}}) to enable fine-grained control of the cache key for improved cache efficiency. For example, the server MAY return the following set of instructions:
 
 ~~~
-  Key: DPR;r=[1.5:]
+  Key: DPR;partition=[1.5:2.5:4.0]
```
Comment (mnot) [inline draft.md]: No surrounding brackets here!

File: draft.md [hunk 2]
```diff
@@ -116,25 +116,25 @@ Above example indicates that the cache key should be based on the DPR header.
...
-  Key: Width;r=[320:640]
+  Key: Width;div=320
 ~~~
 
-Above example indicates that the cache key should be based on the Width header, and the resource should be cached and made available for any request whose display width falls between 320 and 640px.
+Above example indicates that the cache key should be based on the value of the Width header and be partitioned into groups of 320: 0-320, 320-640, and so on.
 
 ~~~
-  Key: Downlink;r=[0.384:]
+  Key: Downlink;partition=[0.5:1.0:3.0:5.0:10]
```
Comment (mnot) [inline draft.md]: No surrounding bracket here!

**Comment by igrigorik** [comment]
@mnot good catch, thanks! Updated, merging.

**Comment by igrigorik** [comment]
@mnot hmm, I can't build with "ietf-httpbis-http-key" reference. Should I still be using the older "fielding-http-key"?

**Comment by mnot** [comment]
It's just `ietf-httpbis-key`.

**Comment by igrigorik** [comment]
Thanks, that did the trick.

**Pull request merged**